### PR TITLE
support pytorch efrc op

### LINKED ISSUE #20577: [Good First Issue]: Support aten::erfc for pytorch models
### Context

OpenVINO component responsible for support of PyTorch models is called as PyTorch Frontend (PT FE). PT FE converts a model represented as TorchScript model to a model in [OpenVINO opset](https://docs.openvino.ai/latest/openvino_docs_ops_opset.html).

### What needs to be done?

- Implement conversion rule and/or transformation to support the new operation.
- Implement operation tests in tests/layer_tests/pytorch_tests. Please consider different data types, but keep reasonable number of test cases

### Example Pull Requests

#18998

### Resources

- [torch.erfc description](https://pytorch.org/docs/stable/generated/torch.erfc.html)
- [What is OpenVINO?](https://github.com/openvinotoolkit/openvino#what-is-openvino-toolkit)
- [OpenVINO PyTorch Frontend](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/pytorch/README.md)
- [Contribution guide](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md)
- [Blog post on contributing to OpenVINO](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md)
- [User documentation](https://docs.openvino.ai/)

### Contact points

@openvinotoolkit/openvino-pytorch-frontend-maintainers 

### Ticket

TBD

### PR DESCRIPTION
**Pull request by ahmadchalhoub**
### Details:
Support aten::erfc op (https://pytorch.org/docs/stable/special.html#torch.special.erfc) for Pytorch in OpenVino.

### Tickets:
- Closes #20577

### CODE REVIEW

File: src/frontends/pytorch/src/op/erfc.cpp [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    if (xdtype.is_dynamic() || !xdtype.is_real()) {
+        x = context.mark_node(std::make_shared<v0::Convert>(x, element::f32));
+    }
+
+    // apply Erf to the input tensor 'x'
+    auto y = context.mark_node(std::make_shared<v0::Erf>(x));
+
+    // create 'ones' to use to calculate complementary of Erf output
+    auto ones = context.mark_node(make_shared<v0::Constant>(element::f32, Shape{}, 1.0f))->output(0);
+    ones = context.mark_node(make_shared<v1::ConvertLike>(ones, y));
```
Comment (mvafin) [inline src/frontends/pytorch/src/op/erfc.cpp:38]:
```suggestion
    auto ones = context.mark_node(make_shared<v0::Constant>(element::f32, Shape{}, 1.0f))->output(0);
    align_eltwise_input_types(context, x, ones);
    
    // apply Erf to the input tensor 'x'
    auto y = context.mark_node(std::make_shared<v0::Erf>(x));
```
Your approach will not work for dynamic case. `align_eltwise_input_types` will align type of FP32 `ones` input and `x` the result will be equal type, which will work for FP64, INT and dynamic case.

**Review by mvafin — CHANGES_REQUESTED** [review]
Please also fix code style

**Review by mvafin — APPROVED** [review]

**Comment by ahmadchalhoub** [comment]
@mvafin hmm looks like code format is still giving errors, even though I checked with clang-format. Any idea why it might be failing?

**Comment by mvafin** [comment]
> @mvafin hmm looks like code format is still giving errors, even though I checked with clang-format. Any idea why it might be failing?

Yes, it is because our codestyle requires 4 spaces instead of 2 spaces. You can run `make clang_format_fix_all` if you had the correct version of clang format installed.

**Comment by mvafin** [comment]
build_jenkins

**Pull request merged**